From: David Cohen <david.a.cohen@intel.com>
Subject: dwc3: increase autosuspend timeout

Current autosuspend timeout is too short. The autosuspend in this case is
useful to avoid a power cycle it an user is unplugging/plugging usb cable in a
short moment. But since current time is 0.5s, it's unlikely the autosuspend
will ever bring any positive side effect.

This patch increases the deadline to 10s to be more realistic.

Signed-off-by: David Cohen <david.a.cohen@intle.com>

---
 drivers/usb/dwc3/core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux.orig/drivers/usb/dwc3/core.c
+++ linux/drivers/usb/dwc3/core.c
@@ -776,7 +776,7 @@ static int dwc3_probe(struct platform_de
 	atomic_set(&dwc->suspend_depth, 0);
 
 	if (dwc->runtime_suspend) {
-		pm_runtime_set_autosuspend_delay(dev, 500);
+		pm_runtime_set_autosuspend_delay(dev, 10000);
 		pm_runtime_use_autosuspend(dev);
 		pm_runtime_mark_last_busy(dev);
 		pm_runtime_put_autosuspend(dev);
